### ISSUE TITLE
Create a branch for FahControl python 2 and master gets python 3

### ISSUE THREAD

**Issue by informatorius** [FoldingAtHome/fah-control#81]
Create a branch for FahControl python 2 so bug fixes can be made there. And let master branch get python 3 so the transition from python 2 can be made there.

**Comment by Hyperspeed1313** [comment]
LONG overdue. Python 2 has been completely EOL'd and removed from Ubuntu 20.04.

**Comment by bb30994** [comment]
Good idea.  The **Long Term Support** distros still contain python 2 for as long as anybody is still running them.

**Comment by roelfrenkema** [comment]
Ubuntu 20.04 no longer supports python-gtk2
